tests/summarycode/data/package_summary/basic-plugin-testing/codebase/src/a.py

```diff
@@ -0,0 +1,6 @@
+# Copyright Example Corp
```

Comment:
Use real examples instead of synthetic ones.

Reply:
Can I use that [`change-case-5.4.4`](https://github.com/blakeembrey/change-case) example here as it has 4 packages inside it and can be more helpful while we have calculated the license_clarity_score since till now we just want to check if the plugin is working?